### ISSUE TITLE
新的数据集

### ISSUE THREAD

**Issue by XingYing-stack** [ayyyq/DORE#2]
您好，感谢您杰出而有启发性的工作！我注意到DocRED数据集中漏标情况严重，并且https://github.com/tonytan48/re-docred 
 提出了重新标注的数据集，请问可以告知在新的数据集下的结果吗？不胜感激！

**Comment by XingYing-stack** [comment]
另外请问目前提供的代码是否仅包含了diagonal采样，并不包含asymmetric采样与parallel row generation? 感谢您的回复！

**Comment by ayyyq** [comment]
> 

您好，很抱歉我最近比较忙，我们会尽快在Re-DocRED上测试我们的模型。

**Comment by ayyyq** [comment]
您好，很抱歉我最近比较忙，我们会尽快在Re-DocRED上测试我们的模型。

**Comment by ayyyq** [comment]
> 您好，感谢您杰出而有启发性的工作！我注意到DocRED数据集中漏标情况严重，并且https://github.com/tonytan48/re-docred 提出了重新标注的数据集，请问可以告知在新的数据集下的结果吗？不胜感激！

您好，很抱歉我最近比较忙，我们会尽快在Re-DocRED上测试我们的模型。

**Comment by ayyyq** [comment]
> 另外请问目前提供的代码是否仅包含了diagonal采样，并不包含asymmetric采样与parallel row generation? 感谢您的回复！

是的。我们发现尽管asymmetric采样和parallel row generation相比窗口大小为1的diagonal采样都有微弱的提升效果，但是不如后者简单高效。经过综合考量，我们大部分实验都只用了窗口大小为1的diagonal采样。

**Comment by XingYing-stack** [comment]
您好，我想仔细调研一下parallel row generation和diagonal采样的区别，请问您能分享一下parallel row generation的代码吗？
